subworkflows/local/call_repeat_expansions/main.nf

```diff
@@ -56,17 +56,16 @@ workflow CALL_REPEAT_EXPANSIONS {
         .groupTuple()
         .set{ ch_bcftools_merge_in }
 
-    BCFTOOLS_MERGE (
+    TRGT_MERGE (
         ch_bcftools_merge_in,
-        ch_fasta,
-        ch_fai,
-        [[],[]]
+        [[],[]],
+        [[],[]],
     )
-    ch_versions = ch_versions.mix(BCFTOOLS_MERGE.out.versions)
+    ch_versions = ch_versions.mix(TRGT_MERGE.out.versions)
 
     emit:
     sample_vcf  = BCFTOOLS_SORT_TRGT.out.vcf  // channel: [ val(meta), path(vcf) ]
-    family_vcf  = BCFTOOLS_MERGE.out.vcf      // channel: [ val(meta), path(vcf) ]
+    family_vcf  = TRGT_MERGE.out.vcf      // channel: [ val(meta), path(vcf) ]
```

Comment:
It would be good to add sample_tbi, and family_tbi here, so we can snapshot these in the tests.